src/main/java/org/terasology/tutorialWorldGeneration/MountainsProvider.java

```diff
@@ -52,12 +54,12 @@ public void process(GeneratingRegion region) {
         ElevationFacet facet = region.getRegionFacet(ElevationFacet.class);
         float mountainHeight = configuration.mountainHeight;
         // loop through every position on our 2d array
-        Rect2i processRegion = facet.getWorldRegion();
-        for (BaseVector2i position : processRegion.contents()) {
+        BlockAreac processArea = facet.getWorldArea();
+        for ( Vector2ic position : processArea) {
```

Comment:
I recommend leveraging the [IntelliJ code optimization feature](https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html)
```suggestion
        for (Vector2ic position : processArea) {
```